### DISCUSSION TITLE
Enable large Bark models in docker container

### DISCUSSION THREAD

**Discussion by EighthOctave** [rsxdalv/TTS-WebUI#139] [Q&A]
I can successfully build and run the docker container (thanks for that by the way!).  I noticed in the logs that the container loads the smaller bark models.  I'd like to use the larger ones since I have the VRAM to support. How do I enable this in the docker container?

I'm a docker novice, BTW.

**Comment by rsxdalv** [comment]
Sorry this message went past me, yes you can do it by editing the config.json (also gets edited when you work with the UI). If you would want a "1-click" deployment for your container, I'd suggest creating a new image with your own custom config.json already created. (By default the config.json gets generated once you launch the UI if it does not exist).

**Reply by rsxdalv** [reply]
So, it depends on _how_ exactly you are using it. If it's for an automated deployment you would need to do some DevOps, if it's for your own container that you reuse, you can just use the settings page and relaunch the container.
